src/main/java/twilightforest/client/renderer/TFMagicMapRenderer.java

```diff
@@ -202,6 +203,30 @@ public void renderMap(EntityPlayer par1EntityPlayer, TextureManager par2TextureM
             GL11.glPopMatrix();
         }
 
+        par2TextureManager.bindTexture(vanillaMapIcons);
+        for (MapCoord mapCoord : magicMapData.featuresVisibleOnMap) {
+            if (mapCoord.iconSize > 80) {
+                GL11.glPushMatrix();
+                GL11.glTranslatef(
+                        (float) var15 + (float) mapCoord.centerX / 2.0F + 64.0F,
+                        (float) var16 + (float) mapCoord.centerZ / 2.0F + 64.0F,
+                        -0.02F);
+                GL11.glRotatef((float) (mapCoord.iconRotation * 360) / 16.0F, 0.0F, 0.0F, 1.0F);
+                GL11.glScalef(4.0F, 4.0F, 3.0F);
+                GL11.glTranslatef(-0.125F, 0.125F, 0.0F);
+                float var21 = (float) (4 % 4 + 0) / 4.0F;
+                float var23 = (float) (4 / 4 + 0) / 4.0F;
+                float var22 = (float) (4 % 4 + 1) / 4.0F;
+                float var24 = (float) (4 / 4 + 1) / 4.0F;
+                tesselator.startDrawingQuads();
+                tesselator.addVertexWithUV(-1.0D, 0D, 0.0D, var21, var23);
+                tesselator.addVertexWithUV(0D, 0D, 0.0D, var22, var23);
+                tesselator.addVertexWithUV(0D, -1.0D, 0.0D, var22, var24);
+                tesselator.addVertexWithUV(-1.0D, -1.0D, 0.0D, var21, var24);
+                tesselator.draw();
+                GL11.glPopMatrix();
+            }
+        }
         // GL11.glPushMatrix();
```

Comment:
Looks good but, can you remove the commented out code?
Just to make the code look cleaner.

Reply:
Done with the cleaning